hack/jenkins/job-configs/kubernetes-e2e.yaml

```diff
@@ -358,13 +358,10 @@
             test-owner: 'beeps'
             provider-env: '{gke-provider-env}'
             job-env: |
-                # XXX Not a unique project
                 export E2E_NAME="e2e-gke-ingress"
```

Comment:
All/Either? I mean why rely on defaults (which are nice, but not as clear if you're trying to figure things out looking at config IMO)